resources/views/site/blocks/3d_embed.blade.php

```diff
@@ -15,13 +17,29 @@
             $cc0 = false;
             break;
     }
+    if (isset($model_size)) {
+        switch ($model_size) {
+            case 0:
+                $size = 's';
+                break;
+            case 1:
+                $size = 'm';
+                break;
+            case 2:
+                $size = 'l';
+                break;
+            default:
+                $size = 'l';
+                break;
+        }
+    }
```

Comment:
Do the numeric values have any utility here? Maybe change the dropdown values in the CMS form to use the letter values directly so you don't have to translate them.

Reply:
@nikhiltri I started with the numerics to add camera_position ratios but since it's not do-able I can change it out in the vue component